docs/concepts/extension_points.rst

```diff
@@ -52,19 +53,19 @@ If you want to provide learners with new content experiences within courses, opt
      - Status
      - Details
    * - **XBlocks**
-     - Adopt, Stable
+     - ✅ **Stable**
      - XBlocks are python plugins specific to Open edX that get installed into edx-platform and can be used to build courses. An XBlock defines a new *type* of interactive component, and authors can then create many instances of that content type in their courses (each with different settings and content). If you only need to support Open edX, XBlocks provide the best user experience. Open edX operators must install an XBlock into their Open edX instance before it can be used. See the `XBlock tutorial`_ to learn more about XBlocks.
    * - **LTI**
-     - Adopt, Stable
+     - ✅ **Stable**
```

Comment:
It may be out of scope for this PR, but saying that "LTI" is stable seems ambiguous. There are multiple LTI implementations in the platform. Which ones are stable? 

The text mentions LTI as a provider, which doesn't have very good documentation, at least. Might that be better described as "Provisional" ? 

Other LTI integrations I am less familiar with: LTI auth, LTI for discussion tools, LTI for proctoring.

Reply:
Good point.  I'm not up to speed on our LTI story, so I was relying on what was already written here. I think I will leave this as is, but try to find someone more knowledgeable to fix this section up. Let me know if you know of any SMEs here.